plugin/evm/config.go

```diff
@@ -219,7 +221,12 @@ func (c Config) EthBackendSettings() eth.Settings {
 	return eth.Settings{MaxBlocksPerRequest: c.MaxBlocksPerRequest}
 }
 
-func (c *Config) SetDefaults() {
+func (c *Config) SetDefaults(ctx *snow.Context) {
+	var trieCleanJournal string
+	if ctx != nil {
+		trieCleanJournal = ctx.ChainDataDir
+	}
```

Comment:
Can we add a suffix to this file path, so that we can re-used the `ChainDataDir` for other use cases in the future as well?

We should also change this from taking in `*snow.Context` to just using `ChainDataDir` so that it takes in the minimal amount of information that is needed here.

Reply:
updated